utils/results/results.go

```diff
@@ -353,23 +435,88 @@ func (ssr *ScaScanResults) HasInformation() bool {
 
 func (ssr *ScaScanResults) HasFindings() bool {
 	for _, scanResults := range ssr.XrayResults {
-		if len(scanResults.Vulnerabilities) > 0 || len(scanResults.Violations) > 0 {
+		if len(scanResults.Scan.Vulnerabilities) > 0 || len(scanResults.Scan.Violations) > 0 {
 			return true
 		}
 	}
 	return false
 }
 
-func (jsr *JasScansResults) GetResults(scanType jasutils.JasScanType) (results []*sarif.Run) {
+func (jsr *JasScansResults) NewApplicabilityScanResults(exitCode int, runs ...*sarif.Run) {
+	jsr.ApplicabilityScanResults = append(jsr.ApplicabilityScanResults, ScanResult[[]*sarif.Run]{Scan: runs, StatusCode: exitCode})
+}
+
+func (jsr *JasScansResults) NewJasScanResults(scanType jasutils.JasScanType, vulnerabilitiesRuns []*sarif.Run, violationsRuns []*sarif.Run, exitCode int) {
```

Comment:
Please rename to AddJasScanResults.
NewJasScanResults, like every func starts with 'New', appears like we are creating new results and here we are adding the new results to the existing struct